### ISSUE TITLE
Backend rerun initial if equation with assert at reinit phase

### ISSUE THREAD

**Issue by OpenModelica-TracImporter** [OpenModelica/OpenModelica#5599]

**Comment by cbu-hw** [comment]
@kabdelhak has this been closed by #6709?

**Comment by kabdelhak** [comment]
> [@kabdelhak](https://github.com/kabdelhak) has this been closed by [#6709](https://github.com/OpenModelica/OpenModelica/pull/6709)?

i assume so, it's 5 years ago and this is imported from trac. Are there still issues with this?

**Comment by cbu-hw** [comment]
No, I just noticed that it's still open although the associated PR has been merged. 
Should have phrased this "I think this can be closed". 😅
